src/main/java/seedu/address/model/MyInsuRec.java

```diff
@@ -110,6 +110,15 @@ public void setClient(Client target, Client editedClient) {
         requireNonNull(editedClient);
 
         clients.setClient(target, editedClient);
+
+        // update meetings associated with the client
+        // by removing all meetings associated with target client
```

Comment:
this is not ideal actually. setclient does one thing and that is to replace target with the editedclient. you should not be performing any thing extra here.

Reply:
this is also buggy in that the meetings that are shifted don't reflect their new owner.

Reply:
in fact thet meetings reflect their new owner, only bc in line 128 of editclientcommand changes it